Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scenes/ObjectManipulatorExample.unity

```diff
@@ -1301,69 +908,10 @@ Transform:
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
   m_LocalPosition: {x: 0, y: 0, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
-  m_Children:
```

Comment:
Feels like a lot of removed `GameObjects` in here, and this children list dropping to nothing.
Is this just a by-product of the updated serialization when you changed the text? Or were there actually removed GOs?

Reply:
that was just a byproduct of saving the scene - i only changed the label text and didn't touch anything else - it's also removing the boundary system / spatial awareness etc.